<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindItemsDialog</class>
 <widget class="QDialog" name="FindItemsDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>347</width>
    <height>138</height>
   </rect>
  </property>
  <widget class="QComboBox" name="searchComboBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>151</width>
     <height>22</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="editable">
    <bool>true</bool>
   </property>
   <property name="insertPolicy">
    <enum>QComboBox::InsertAtTop</enum>
   </property>
  </widget>
  <widget class="QCheckBox" name="regexCheckBox">
   <property name="geometry">
    <rect>
     <x>119</x>
     <y>84</y>
     <width>115</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Regular expression</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="caseSensitiveCheckBox">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>61</y>
     <width>92</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Case sensitive</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="exactMatchCheckBox">
   <property name="geometry">
    <rect>
     <x>119</x>
     <y>61</y>
     <width>82</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Exact match</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="wrapAroundCheckBox">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>84</y>
     <width>86</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Wrap around</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="previousButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>261</x>
     <y>40</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Find previous</string>
   </property>
  </widget>
  <widget class="QPushButton" name="searchResultsButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>261</x>
     <y>69</y>
     <width>76</width>
     <height>23</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Search results</string>
   </property>
  </widget>
  <widget class="QPushButton" name="closeButton">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>100</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Close</string>
   </property>
  </widget>
  <widget class="QPushButton" name="nextButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>261</x>
     <y>11</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string>Find next</string>
   </property>
   <property name="default">
    <bool>true</bool>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="searchPropsCheckBox">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>110</y>
     <width>119</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Search in properties</string>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>closeButton</sender>
   <signal>clicked()</signal>
   <receiver>FindItemsDialog</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>407</x>
     <y>121</y>
    </hint>
    <hint type="destinationlabel">
     <x>348</x>
     <y>116</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
